spread.yaml

```diff
@@ -230,6 +231,7 @@ backends:
                   password: ubuntu
 
     qemu:
+        memory: 4G
```

Comment:
I guess that these changes were committed here by mistake?

Reply:
No this was intentional - with the changes to the snapd-snap spread test, a local qemu VM needs more memory to be able to execute it successfully.